src/api/student.js

```diff
@@ -49,4 +49,20 @@ app.post('/', (req, res) => {
     }
     res.successJson();
 });
+
+app.patch('/item', (req, res) => {
+    if (typeof req.query.studentId !== 'string') {
```

Comment:
Hi. This is a query string therefore it will always come in as a string. I have a parseInt underneath of the check after confirming it exists in the req.